nautobot/dcim/templates/dcim/inc/cable_termination.html

```diff
@@ -20,6 +20,20 @@
                 <a href="{{ termination.get_absolute_url }}">{{ termination }}</a>
             </td>
         </tr>
+    {% elif termination.power_panel %}
+        {# Device Power Panel #}
+        <tr>
+            <td>Power Feed</td>
+            <td>
+                <a href="{{ termination.get_absolute_url }}">{{ termination }}</a>
+            </td>
+        </tr>
+        <tr>
+            <td>Power Panel</td>
```

Comment:
Sorry to nitpick, but looking at the order of the other options (Device before component, provider before circuit) I think it would be more consistent to order power panel before power feed.